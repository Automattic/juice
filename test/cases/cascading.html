<html><body>
<ul>
  <li>
    <ul>
      <li>
        <ul>
          <li></li>
        </ul>
      </li>
    </ul>
  </li>
</ul>
</body></html>
