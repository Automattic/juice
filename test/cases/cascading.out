<html><body>
<ul>
  <li style="display: block; color: #f00;">
    <ul>
      <li style="display: block; color: #f00;">
        <ul>
          <li style="display: block; color: #f00;"></li>
        </ul>
      </li>
    </ul>
  </li>
</ul>
</body></html>
